src/database/queries/test/test.js

```diff
@@ -0,0 +1,53 @@
+const test = require('tape');
+const runDbBuild = require('../../db_build');
+const { setBook } = require('../set_book');
+const { setCategory } = require('../set_category');
+const { getCategory } = require('../get_category');
+
+test('Test for the getCategory function', (t) => {
+  runDbBuild('db_bulid.sql', (err, res) => {
+    t.notOk(err);
+    return runDbBuild('fake_data.sql', () => {
+      getCategory()
+        .then((response) => {
+          t.equal(response.length > 0, true, 'getCategory returns data successfully ');
+          t.end();
+        })
+        .catch(error => t.error(error));
+    });
+  });
+});
+
+test('Test for the setBook function', (t) => {
+  runDbBuild('db_bulid.sql', (err, res) => {
+    t.notOk(err);
+    return runDbBuild('fake_data.sql', () => {
+      const data = {
+        nameBookVal: 'يوسف يامريم', nameAuthorVal: 'يامى أحمد', imageBookVal: '', descriptionVal: 'يوسف يامريم', categorySerial: '503',
+      };
+      setBook(data)
+        .then((response) => {
+          t.equal(typeof response, 'object', 'setBook returns data successfully ');
```

Comment:
if you logged your `response` you will find that it is an empty `object`
so your test always will pass
**be specific to what are you testing**